docs/api-reference.md

```
@@ -1126,46 +1132,46 @@ topk(100, openqoe_metric{org_id="org_abc123"})
+| `network` | `TIMEOUT`                | Request timeout               |
+| `decoder` | `MEDIA_DECODE_ERROR`     | Failed to decode media        |
+| `decoder` | `UNSUPPORTED_CODEC`      | Codec not supported           |
+| `source`  | `INVALID_MANIFEST`       | Manifest parsing error        |
+| `source`  | `INVALID_MEDIA`          | Invalid media format          |
+| `drm`     | `LICENSE_REQUEST_FAILED` | DRM license request failed    |
+| `drm`     | `KEY_SYSTEM_ERROR`       | Key system error              |
 
 ---
 
-**End of API Reference v1.0**
+**End of API Reference v2.0**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)